teps/0074-deprecate-pipelineresources.md

```diff
@@ -2,7 +2,7 @@
 status: proposed
 title: Deprecate PipelineResources
 creation-date: '2021-07-14'
-last-updated: '2021-11-05'
+last-updated: '2022-02-25'
```

Comment:
btw feel free to add yourself as an author if you want! 😉

Reply:
Thanks! I will if I end up making some more substantial changes here :)